docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,146 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+**coala** has an awesome testing framework so you can write tests for bears
+with ease.
+
+You can use the following to test your bears:
+
+- ``LocalBearTestHelper.check_validity``
+- ``LocalBearTestHelper.check_results``
+- ``verify_local_bears``
+
+Understanding through examples
+------------------------------
+
+**EXAMPLE 1** using ``verify_local_bear``
+
+PyDocStyleBearTest
```

Comment:
you can just write "MyBear" for example everywhere, and for the good file contents you write some text like "This is considered good code", same principle for bad code^^